src/DwitTech.NotificationService.Data/Entities/Email.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.NotificationService.Data.Entities
+{
+    public class Email : BaseEntity
+    {
+        public int Id { get; set; }
```

Comment:
I believe the BaseEntity already defines an Id so this field shouldnt be here.

Reply:
> I believe the BaseEntity already defines an Id so this field shouldnt be here.

Resolved